init.go

```diff
@@ -67,8 +69,11 @@ func (b *command) start() error {
 	}
 }
 
+// With these separation we do not need AppLogger plugin anymore. Just write logs to stdout/stderr
 func (b *command) Write(data []byte) (int, error) {
-	b.log.Info(string(data))
+	// All output from the application does not intersect with logs from the Server plugin
+	// For example: destroy signal received	{"timeout": 60000000000} is not necessary for logging
+	b.appLog.Error(string(data))
```

Comment:
This is a fairly simple task, it is solved out of the box in [PHP-FPM](https://www.php.net/manual/en/install.fpm.configuration.php), see `catch_workers_output`.

We just need to separate the application output from the server and nothing more. No plugins or additional settings are needed for this. In essence, with this feature we get rid of the AppLogger plugin. I think this plugin was created precisely because there was no easy way to separate application logs. The level of info is quite suitable.

I still don't understand: what needs to be done as in the "temporal" plugin. The main thing is why?

Reply:
If you want to match with `catch_workers_output` it should be the same option and **raw** output to the main log (file, stdout or stderr). You can't just change the log to be shown as errors. 
AppLogger was not created to handle this case, this is a completely separate feature which was requested by users to solve other needs.

Reply:
"this is a completely separate feature" look at this:

AppLogger: send raw logs to `app` channel from app by RPC.

This feature: send raw logs to `app` channel from app by stdout/err and config:
```
logs:
  channels:
     app:
       level: debug
       mode: raw
```

With a little changes, you can do the same thing that the AppLogger plugin does, but much easier: without RPC, plugin, [app-logger php package](https://github.com/roadrunner-php/app-logger) and additional packages for [Monolog](https://github.com/Seldaek/monolog) and other loggers.

Is there any other reason for the AppLogger's existence?

Reply:
App logger was requested by PHP devs and advised to include by PHP devs with. This plugin won't be removed. For the reasoning, please, use GitHub search, this is an OSS project, all discussions about all features are open for everyone.

Returning to the original issue:
Like you mentioned for the applogger, the same you can achieve with the `server` logger channel for your proposed change with info log level.

```yaml
logs:
  channels:
     server:
       level: info
       mode: raw
```

Your `appLogger` info log (in the PR) won't solve any problem here, because it'd be hidden from the user in the same way the current `server` log is hidden when not configured to show info logs from the server plugin.

Reply:
I did not offer to remove the AppLogger plugin, I only showed its optionality. I use it myself and is forced to additionally configure the logger through the plugins. Cool, if you can solve the problem only with the help of configuration without writing code.

I suggest not just add some hidden `app` channel. I propose an idea that can be implemented in the future release (maybe 2025).

Idea: There is always an `app` channel and it always comes out of the PHP-processes. It is configured in a certain way out of the box. This behavior is superior and does not scare users.

Reply:
I guess this is a good idea, but it should not be connected to the existing logger then. It should be controlled by the similar to FPM options (?), like `catch_workers_output` and just output to stderr/stdout what the worker sent.